### DISCUSSION TITLE
New to ZigbeeWorld and new Bulbs from AWOX

### DISCUSSION THREAD

**Discussion by 303acid** [Koenkk/zigbee2mqtt#10004] [General]
Hi,
I'm complete new on this redmatic and zigbee world (but not in Linux!) and need some help.
I have a HomeMatic CCU3 with cc2531 USB-Stick and Redmatic. The Stick is working and i have 'paired' my two AWOX bulb's.
And now? How can i use this items? There are so many new thinks for me - i can't see the forest because there are to many tree's

can anyone give me some tips and hints how this radio magic is working?

btw: 
AWOX has a super tuber mobile App where you MUST have a awox-account, a working internet connection AND GPS enabled to pair or control the light with your mobile. and all this is not printed on the package...suckers!

**Comment by tromanski** [comment]
Hello @303acid,

I have issues to configure my CC2531 on my CCU3 with raspberrymatic and redmatic. Can you tell me how to configure the stick so it will bee seen by zivbee2mqtt inside redmatic? 

Thank you in advance

**Comment by 303acid** [comment]
hi,

i moved with the CC2531 from CCU3 to a archlinux_x86_64 with redmatic
zigbee2mqtt and mosquitto.
but it should also work on a ccu3.
in nodered i use the plugin node-red-contrib-zigbee2mqtt....

so for simplifying, i will give you my simple home flow for lights,
temp/humidity, and heater valve. the heater valve is read/set over the
ccu3-plugins. the temp/humidity sensor is zigbee and set give the heater
valve response ;-). the 'Taster_WHZ' is a zigbee switch with 6 buttons
maybe you can see how it's work

Am Fr., 16. Juni 2023 um 10:08 Uhr schrieb tromanski <
***@***.***>:

**Reply by tromanski** [reply]
hi,

thank you for your first feedback. Can you send me the flow please?
According the instruction for zigbee2mqtt the configuration has to be made in the configuration.yaml. Can you tell me where this file is or isn´t that necessary int his case?

**Comment by 303acid** [comment]
the flow was already send as attachment...? i will send it again.
i can't say how you configure zigbee2mqtt on a ccu3, sorry.

Am Fr., 16. Juni 2023 um 22:25 Uhr schrieb tromanski ***@***.***>:
